mllib/src/test/scala/org/apache/spark/ml/classification/GBTClassifierSuite.scala

```diff
@@ -354,6 +356,41 @@ class GBTClassifierSuite extends SparkFunSuite with MLlibTestSparkContext
   }
 
   /////////////////////////////////////////////////////////////////////////////
+  // Tests of feature subset strategy
+  /////////////////////////////////////////////////////////////////////////////
+  test("Tests of feature subset strategy") {
+    val numClasses = 2
+    val gbt = new GBTClassifier()
```

Comment:
Not sure if we need these tests. We already test the `featureSubsetStrategy` in the RandomForest implementation. At any rate, let's not set parameters that aren't strictly relevant to the test (stepSize, impurity, etc...).

Reply:
Removed stepSize , impurity and other  parameters

Reply:
Setting maxIter and maxDepth to some small value is actually useful so the tests don't take unnecessarily long.